### ISSUE TITLE
Identification of group for different package types

### ISSUE THREAD

**Issue by ajinkya-harness** [Vity01/backstage-jfrog-artifactory-libs#10]
Hi,
Is there any check list where we can identify when to use the group annotation? It is a bit unclear when exactly we have to add a group annotation, for example in terraform package type. Also, can we have some clarity on how to identify group of a package in docs?

TIA

**Comment by Vity01** [comment]
No, unfortunately it's not. It depends how the artifact is stored in the artifactory and it's different for each storage/repository type.
